ublksrv_tgt.cpp

```diff
@@ -1000,7 +1000,7 @@ int main(int argc, char *argv[])
 
 	char *cmd;
 	int ret;
-	char exe[PATH_MAX];
+	char exe[PATH_MAX + 1];
```

Comment:
#define PATH_MAX        4096    /* # chars in a path name including nul */

So here one extra byte isn't needed.